common/lib/capa/capa/capa_problem.py

```diff
@@ -950,21 +950,26 @@ def response_a11y_data(self, response, inputfields, responsetype_id, problem_dat
 
                 label = inputfields[0].attrib['label']
                 # Get first <p> tag before responsetype, this <p> contains the question text.
-                p_tag = response.xpath('preceding-sibling::p[1]')
+                p_tag_list = response.xpath('preceding-sibling::p[1]')
+                try:
+                    p_tag = p_tag_list[0]
+                except IndexError:
+                    p_tag = None
```

Comment:
`p_tag = p_tag_list and p_tag_list[0]`

`p_tag` will be set to empty list or first element in list if present.